crates/shared/src/http_solver.rs

```diff
@@ -108,6 +106,10 @@ impl HttpSolverApi for DefaultHttpSolverApi {
                 use_internal_buffers.to_string().as_str(),
             );
         }
+        if let Some(auction_id) = maybe_auction_id {
+            url.query_pairs_mut()
+                .append_pair("auction_id", auction_id.to_string().as_str());
```

Comment:
Does the MIP solver support this parameter?

Reply:
Not MIP (which is being deprecated btw), not sure about cowdex.

Reply:
Can we merge this?

Reply:
Should this also use the same fallback value for `auction_id` as the instance name so that the query parameter always gets added?

Reply:
Cow does not yet. But after I get an approval in this PR, it should be good: 
https://github.com/gnosis/cow-dex-solver/pull/18